LayoutTests/inspector/console/js-completions.html

```diff
@@ -52,6 +52,27 @@
         }
     });
 
+    suite.addTestCase({
+        name: "console.jsCompletions.completionOrdering",
+        description: "Test that the JavaScript completions are ordered correctly.",
+        async test() {
+            await InspectorTest.evaluateInPage(`myObject = {
```

Comment:
if you truly want to have full control over what's in the object you could do `Object.create(null)` as that'll prevent any of the default stuff :)